WooCommerce/Classes/POS/Presentation/Card Present Payments/Reader Messages/PointOfSaleCardPresentPaymentProcessingMessageViewModel.swift

```diff
@@ -15,7 +15,7 @@ private extension PointOfSaleCardPresentPaymentProcessingMessageViewModel {
 
         static let pleaseWait = NSLocalizedString(
             "pointOfSale.cardPresent.paymentProcessing.message",
-            value: "Please wait...",
+            value: "Please wait",
```

Comment:
We need to update the key here, otherwise the string value won't be updated in the UI:

![Simulator Screenshot - iPad Air 11 - iOS 17 5 M2 (US site) - 2025-02-20 at 08 13 27](https://github.com/user-attachments/assets/8f78f828-c303-41e6-84bd-31f922e1a625)

Eg:
```
pointOfSale.cardPresent.waitingForPaymentProcessing.message
```

Reply:
(Sorry I commented rather than starting a review, still ongoing!)

Reply:
Thanks for catching this, I was hoping the string value just cached on my end but it looks like the string key has to change for the new value to work. Updated in https://github.com/woocommerce/woocommerce-ios/pull/15209/commits/146ca3c887bfab7456992658a57f7a36d27799ec

Reply:
<img src="https://github.com/user-attachments/assets/d7295172-735f-44ee-999a-6dff9e2e51d9" width="500" />